### PR TITLE
New GuitarChord Cheet Sheet

### PR DESCRIPTION
**Pull request by binilj04**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

-  New Instant Answer
    - Cheat Sheets:  **`New GuitarChord Cheet Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Provide an overview of the changes this pull request introduces.

###### Which issues (if any) does this fix?

Fixes #NNNN - how specifically does it fix the issue?

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/guitar_chord

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
@binilj04 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/guitarchord.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
+{
+    "id": "guitarchord_cheat_sheet",
+    "name": "Guitar Chord",
+    "description": "The first column represents the guitar string open notes with the first string (high E) on top. The number on top row are the Fret numbers.The numbers written between the frets represent the finger numbers: Index Finger =1  to Little Finger=4. 0 finger meaning open string and x meaning not to touch or making it mute.",
```
Comment (adityaef) [inline share/goodie/cheat_sheets/json/guitarchord.json]: Consider removing period at the end

File: share/goodie/cheat_sheets/json/guitarchord.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    "metadata": {
+        "sourceName": "Guitar Chords",
```
Comment (gaulrobe) [inline share/goodie/cheat_sheets/json/guitarchord.json]: This should be `8notes.com` since that is the name of the source website.

**Comment by gaulrobe** [comment]
Taking another look at this... we already have a Goodie that provides guitar chord info: https://duckduckgo.com/?q=Am+guitar+chord&ia=answer

**Comment by alohaas** [comment]
@binilj04 thanks for this thoughtful addition to our Instant Answers, though as @gaulrobe mentioned, we already have a cheat sheet for guitar chords!

Can you think of any other guitar or music-related Anstant Answers that you would like to create instead? You can search for what we already have here: [https://duck.co/ia](https://duck.co/ia)

Let us know if you have any questions or ideas. If it's alright with you, I'm going to close this PR.

**Comment by binilj04** [comment]
@alohaas, yes please close this PR. I got what i was looking for and it is beautifully done

**Pull request closed without merging**